packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Allows us to monkey patch Zone prototype without TS compiler errors.
+declare const Zone: ZoneType & { prototype: Zone };
+
+export interface AsyncTaskData {
+  interactionId: string;
+  pageView: string;
+}
+
+export type AsyncTask = Task & {
+  data: AsyncTaskData;
+  eventName: string;
+  target: HTMLElement;
+};
+
+export class InteractionTracker {
+  constructor() {
+    const runTask = Zone.prototype.runTask;
+    Zone.prototype.runTask = function(
+      task: AsyncTask,
+      // tslint:disable-next-line:no-any
+      applyThis: any,
```

Comment:
Could you change this to `unknown` here and below and then remove the tslint error?